### DISCUSSION TITLE
Trouble installing it on 1.19.2

### DISCUSSION THREAD

**Discussion by Julu2106** [eccentricdevotion/TARDIS#732] [Q&A]
Hi, I just started upgrading my server to 1.19.2 and can't get the tardis plugin running. I moved it into the "plugins" folder as I always used to do.  Server is running on craftbukkit. Any help's deeply appreciated.

**Comment by Chew** [comment]
Are you using the jars from the TARDIS from the https://github.com/eccentricdevotion/TARDIS/releases/tag/1.19.2 ?

**Comment by eccentricdevotion** [comment]
TARDIS has not supported _CraftBukkit_ servers for some years, please use **Spigot** or **Paper**. I personally use Paper.
You can build Spigot yourself or download it here: http://tardisjenkins.duckdns.org:8080/job/BuildTools/lastSuccessfulBuild/artifact/spigot-1.20.1.jar
Paper can be downloaded here: https://api.papermc.io/v2/projects/paper/versions/1.19.2/builds/307/downloads/paper-1.19.2-307.jar
